.travis.yml

```diff
@@ -16,16 +16,20 @@ script:
 - docker-compose run --rm manage.py fetch_csv
 - docker-compose run --rm manage.py import_reqs data.csv
 before_deploy:
-  - export PATH=$HOME:$PATH
-  - travis_retry curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github" | tar -zx
-  - travis_retry wget "https://github.com/contraband/autopilot/releases/download/0.0.3/autopilot-linux"
-  - chmod a+x autopilot-linux
-  - mv cf $HOME
-  - mv autopilot-linux $HOME
-  - cf install-plugin -f ~/autopilot-linux
+- export PATH=$HOME:$PATH
+- travis_retry curl -L "https://cli.run.pivotal.io/stable?release=linux64-binary&source=github"
+  | tar -zx
+- travis_retry wget "https://github.com/contraband/autopilot/releases/download/0.0.3/autopilot-linux"
+- chmod a+x autopilot-linux
+- mv cf $HOME
+- mv autopilot-linux $HOME
+- cf install-plugin -f ~/autopilot-linux
 deploy:
   provider: script
   script: devops/deploy.sh dev
   skip_cleanup: true
   on:
     branch: master
+after_success:
+- docker-compose run --rm dev npm install codeclimate-test-reporter
```

Comment:
I'm not sure if this makes things more complicated, but we have a docker-compose command/service for `npm`, meaning these two commands are almost identical:
```
docker-compose run --rm npm install codeclimate-test-reporter
docker-compose run --rm dev npm install codeclimate-test-reporter
```

The difference is that the first runs `install codeclimate-test-reporter` in the `npm` container (which prefixes everything with `npm `) while the second runs `npm install codeclimate-test-reporter` in the `dev` container (which doesn't have any command prefix).

No requested changes, just hoping to clear up any confusion.